test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems

```diff
@@ -162,6 +162,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)\LeakTestPage.xaml.cs" Condition="$(SolutionName) != 'MUXControlsInnerLoop'">
       <DependentUpon>LeakTestPage.xaml</DependentUpon>
     </Compile>
+    <Compile Include="$(MSBuildThisFileDirectory)\InfoBarTestPage.xaml.cs" Condition="$(SolutionName) != 'MUXControlsInnerLoop'">
```

Comment:
can we add an InfoBar folder similar to the other controls and create an InfoBar.Shared.proj file ?

Reply:
I think we can use the createnewcontrolproject script to create the scaffolding and go with that as a starting point.

Reply:
Sure! I created a shared proj folder, and all of my infobar files are underneath dev/infobar now.